src/bitcast.rs

```diff
@@ -1,32 +1,8 @@
-// Ensure that the source and destination types are both primitive integer
-// types and the same size, and then bitcast.
-#[allow(unused_macros)]
-macro_rules! bitcast {
-    ($x:expr) => {{
-        if false {
-            // Ensure the source and destinations are primitive integer types.
-            let _ = !$x;
-            let _ = $x as u8;
-            0
-        } else if false {
-            // Ensure that the source and destinations are the same size.
-            // SAFETY: This code is under an `if false`.
-            #[allow(unsafe_code, unused_unsafe, clippy::useless_transmute)]
-            unsafe {
-                ::core::mem::transmute($x)
-            }
-        } else {
-            // Do the conversion.
-            $x as _
-        }
-    }};
-}
```

Comment:
It's unclear how this change improves theoretical safety, since this `bitcast` macro does not execute unsafe code, while zerocopy::transmute [does execute unsafe code](https://docs.rs/zerocopy/latest/src/zerocopy/lib.rs.html#1409).

Reply:
Ah I misread the macro; you're right that zerocopy doesn't add anything unsafe-wise; removed.